functions.php

```diff
@@ -171,10 +173,42 @@ function judge_child_customizer( WP_Customize_Manager $wp_customize ) {
 function show_author_info() {
 	if ( is_plugin_active( 'external-author/external-author.php' ) ) {
 		if ( get_post_meta( get_the_ID(), '_external_authors_no_author', true ) ||
-			count( get_post_meta( get_the_ID(), '_external_authors', true ) ) > 0
+		     count( get_post_meta( get_the_ID(), '_external_authors', true ) ) > 0
```

Comment:
"WebStorm inserts tabs for indentation and reformatting, but fine alignment to a necessary column is done only via spaces without replacing them with tabs. This is done in order to preserve visual representation of the source code, when the Tab Size is changed." =)
A tab would move the condition further right and make it look worse.